.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.x
+        tools: composer:v1
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate --strict
+
+    - name: Cache Composer packages
+      id: composer-cache
+      uses: actions/cache@v2
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
```

Comment:
Esse passo aqui parece mágica. Imagino que tenha vindo de algum how-to, ou mesmo do peaceiris/actions-gh-pages? De qualquer forma, é bom indicar a origem (desse comando, ou das actions todas msm), caso precise de manutenção no futuro.

Também tá faltando atualizar o Readme 😅

Reply:
Sim rsrs. Veio do starter workflow de PHP que o GitHub oferece na hora de criar um workflow:
![image](https://user-images.githubusercontent.com/7695608/129607322-2cdc2fb5-0734-4f38-bc94-9b09f7008a78.png)
https://github.com/actions/starter-workflows/blob/e3ebfcb93eb466e50c9a2ee7705624706e80672b/ci/php.yml

Esse step *cacheia* a pasta vendor para que não seja necessário baixar as dependências novamente a cada run se não houver mudanças.

Eu posso atualizar o Readme e adicionar comentários no arquivo do workflow também explicando melhor o que cada passo faz. O que acha?

Reply:
O nome da action é normalmente a organização/action do próprio github.

Volta e meia vou no source para entender qual é a magia que a action faz.

Exemplo: 

https://github.com/actions/cache

Reply:
De fato vale um comentário no readme informando que o deploy é feito automaticamente usando github actions

Reply:
Atualizei o README e o arquivo do workflow.